### PR TITLE
Support operator APIs

### PR DESCRIPTION
**Pull request by jianoaix**
## Why are these changes needed?
Implementing dataapi v2 (https://github.com/Layr-Labs/eigenda/pull/955)

## Checks

- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- [x] I've checked the new test coverage and the coverage percentage didn't drop.
- Testing Strategy
   - [x] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: disperser/dataapi/server_v2.go [hunk 1]
```diff
@@ -245,16 +251,96 @@ func (s *ServerV2) FetchBatchHandler(c *gin.Context) {
...
+//	@Param		operator_id	query		string	false	"Operator ID in hex string [default: all operators if unspecified]"
+//	@Success	200			{object}	OperatorsStakeResponse
+//	@Failure	400			{object}	ErrorResponse	"error: Bad request"
+//	@Failure	404			{object}	ErrorResponse	"error: Not found"
+//	@Failure	500			{object}	ErrorResponse	"error: Server error"
+//	@Router		/operators/stake [get]
 func (s *ServerV2) FetchOperatorsStake(c *gin.Context) {
-	errorResponse(c, errors.New("FetchOperatorsStake unimplemented"))
+	timer := prometheus.NewTimer(prometheus.ObserverFunc(func(f float64) {
+		s.metrics.ObserveLatency("FetchOperatorsStake", f*1000) // make milliseconds
```
Comment (dmanc) [inline disperser/dataapi/server_v2.go:267]: These can be replaced with grpc interceptor?

**Review by dmanc — APPROVED** [review]

File: disperser/dataapi/server_v2.go (hunk 1 above)
Comment (dmanc) [reply disperser/dataapi/server_v2.go:267]: Oh wait dataapi is http, but I think similar concept might apply?

Comment (dmanc) [reply disperser/dataapi/server_v2.go:267]: https://github.com/Depado/ginprom

Comment (jianoaix) [reply disperser/dataapi/server_v2.go:267]: It looks a hobby project, otherwise we could drop these primitive level metrics in application layer.

**Pull request merged**